src/test/java/core/basesyntax/operation/impl/OperationServicePurchaseImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.operation.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.OperationService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationServicePurchaseImplTest {
```

Comment:
make test for case if in shop no a fruit that person want to buy

Reply:
allready - apply_fruitTransactionWithNoInfoAboutFruit_notOk();